spec/migrations/20210521192754_update_product_features_where_identifier_was_renamed_since_lasker_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_migration
+
+class UpdateProductFeaturesWhereIdentifierWasRenamedSinceLasker
+  class MiqRolesFeature < ActiveRecord::Base; end
+end
+
+describe UpdateProductFeaturesWhereIdentifierWasRenamedSinceLasker do
+  let(:user_role_id) { anonymous_class_with_id_regions.id_in_region(1, anonymous_class_with_id_regions.my_region_number) }
+  let(:feature_stub) { migration_stub :MiqProductFeature }
+  let(:roles_feature_stub) { migration_stub :MiqRolesFeature }
+
+  migration_context :up do
+    describe 'product features update' do
+      it "renamed features aren't removed from user roles" do
+        features = {}
+        UpdateProductFeaturesWhereIdentifierWasRenamedSinceLasker::FEATURE_MAPPING.each do |before, _|
```

Comment:
super-de-dooper minor, but this can be `described_class::FEATURE_MAPPING` to save yourself from the long class names.